### PR TITLE
Bump k8s Go dependencies to v0.19.0

### PR DESCRIPTION
**Pull request by xrstf**
**What this PR does / why we need it**:
It's time to update to a stable release. The rc.3 was from when we did the big Go Modules refactoring andwanted to use the latest because it made the dependency tree easier.

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```

### CODE REVIEW

File: hack/lib.sh [hunk 1]
```diff
@@ -266,14 +266,14 @@ check_all_deployments_ready() {
   local deployments
   deployments=$(kubectl -n $namespace get deployments -o json)
 
-  if [ $(jq '.items | length' <<< $deployments) -eq 0 ]; then
```
Comment (kron4eg) [inline hack/lib.sh]: IIRC, this is the way shellcheck is insisting it needs to be. Do we want to drop it?

Comment (xrstf) [reply hack/lib.sh]: Fixed

**Review by kron4eg — APPROVED** [review]

**Pull request merged**